templates/common/fastapi/src-pyloid/build/macos_optimize.spec

```
@@ -0,0 +1,36 @@
+Contents/Frameworks/PySide6/Qt/lib/QtSensors*
+Contents/Frameworks/PySide6/Qt/lib/QtSerialBus*
+Contents/Frameworks/PySide6/Qt/lib/QtSerialPort*
+Contents/Frameworks/PySide6/Qt/lib/QtShaderTools*
+Contents/Frameworks/PySide6/Qt/lib/QtSpatialAudio*
+Contents/Frameworks/PySide6/Qt/lib/QtSvg*
+Contents/Frameworks/PySide6/Qt/lib/QtTextToSpeech*
+Contents/Frameworks/PySide6/Qt/lib/QtVirtualKeyboard*
+Contents/Frameworks/PySide6/Qt/lib/QtTest*
+Contents/Frameworks/PySide6/Qt/qml
+Contents/Frameworks/PySide6/Qt/lib/QtWebEngineCore.framework/Versions/A/Resources/qtwebengine_devtools_resources.pak
+Contents/Frameworks/PySide6/Qt/lib/QtWebEngineCore.framework/Versions/A/Resources/qtwebengine_locales/*.pak
```

RuboCop (1.87.0)

[fatal] unexpected token tSTAR
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)